github/repos_test.go

```diff
@@ -42,15 +42,17 @@ func TestRepositoriesService_List_specifiedUser(t *testing.T) {
 		testMethod(t, r, "GET")
 		testHeader(t, r, "Accept", mediaTypeLicensesPreview)
 		testFormValues(t, r, values{
-			"type":      "owner",
-			"sort":      "created",
-			"direction": "asc",
-			"page":      "2",
+			"visibility":  "public",
+			"affiliation": "owner,collaborator",
+			"type":        "owner",
```

Comment:
Type is documented as:

> Will cause a 422 error if used in the same request as visibility or affiliation.

So this is not a very realistic scenario. I know it's mock values and will not actually cause 422, but it's probably still better to simulate a real request.